integration-tests/client-rpc/test/network-ops.test.ts

```diff
@@ -163,6 +163,77 @@ describe("Staking", () => {
 		};
 	};
 
+	const testTransferAndStakeAmountBaseUnit = async (
```

Comment:
I think you forgot to create a test and run this function

Reply:
you are right, I'll fix it, thanks